app/components/article-featured-video.js

```diff
@@ -24,6 +24,46 @@ export default Component.extend(
 			this._super(...arguments);
 
 			this.initVideoPlayer();
+			this.initOnScrollBehaviour();
+		},
+
+		/**
+		 * Manages video transformation on user's scroll action
+		 *
+		 * @returns {void}
+		 */
+		initOnScrollBehaviour() {
+			let $video = this.$('.video-container'),
+				$siteHead = $('.site-head'),
+				videoBottomPosition = $video.offset().top + $video.height(),
+				showVideoOnScroll = true;
+
+			$(window).on('scroll', {player: this.player, $window: this.$(window)}, function ({data: {player, $window}}) {
+				run.throttle(this, function () {
+					let currentScroll = $window.scrollTop();
+
+					if (currentScroll >= videoBottomPosition && !$video.hasClass('fixed')) {
+						if (showVideoOnScroll && (player === undefined || ! player.isPlaying())) {
+							$video.addClass('fixed');
+							$siteHead.addClass('no-shadow');
+						}
+					} else if (currentScroll < videoBottomPosition - $video.height() && $video.hasClass('fixed')) {
+						$video.removeClass('fixed');
+						$siteHead.removeClass('no-shadow');
+					}
+				}, 200);
+			});
+
+			$video.find('.video-close-button').on('click', function () {
```

Comment:
Please covert this function to Ember action: https://guides.emberjs.com/v2.12.0/templates/actions/